addons/hr_recruitment_survey/tests/test_recruitment_survey.py

```diff
@@ -46,7 +46,6 @@ def test_send_survey(self):
         action = self.job_sysadmin.action_send_survey()
 
         invite_form = Form(self.env[action['res_model']].with_context({
-            'active_id': self.job_sysadmin.id,
```

Comment:
I see that you already have a test here that checks that the "action_invite" correctly creates the answer.
Meaning there is probably no need to add an extra one.
Maybe you could add a check that the "response_id" field of the applicant matches the retrieved survey.user_input?